.github/workflows/experimental.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby: [head, jruby-head, truffleruby, truffleruby-head]
+        ruby: [head, jruby-head, truffleruby-head]
```

Comment:
@eregon surely it's in either one or the other but not both?

Reply:
The idea is it'd be `truffleruby` again in `ci.yml` once that's good enough to pass CI, in my understanding this is just a draft PR to get any version of TruffleRuby to pass the CI.